test/lint/lint-assertions.py

```diff
@@ -12,7 +12,7 @@
 
 def git_grep(params: [], error_msg: ""):
     try:
-        output = subprocess.check_output(["git", "grep", *params], universal_newlines=True, encoding="utf8")
+        output = subprocess.check_output(["git", "grep", *params], text=True, encoding="utf8")
```

Comment:
I think `encoding` is now unnecessary to specify in this line (and many others below) because `text` ensures stdout is a string and strings in Python 3 are by default unicode.

Reply:
Heh, no. This only changed in Python 3.7, see https://docs.python.org/3/library/os.html#utf8-mode

I guess `text`/`universal_newlines` is not needed when specifying `encoding`, so might as well remove them completely.

Reply:
Why wouldn't they be needed? Windows still has non-standard newlines.